src/main/java/com/ankist/expensetracker/controller/ExpenseRestService.java

```diff
@@ -0,0 +1,46 @@
+package com.ankist.expensetracker.controller;
+
+import com.ankist.expensetracker.model.ExpenseDTO;
+import com.ankist.expensetracker.repository.ExpenseDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class ExpenseRestService {
+
+    @Autowired
+    private ExpenseDAO expenseServiceDAO;
+
+
+    @GetMapping("/getById")
+    public ResponseEntity<ExpenseDTO> findById(@RequestParam("id") Integer id){
```

Comment:
Using verbs in API endpoints is not encouraged. Use path variable instead. Using verbs in API endpoints is not encouraged. Use path variable instead. Also please avoid camelCase in API endpoints. Requested an API request mapping change. You can rename this endpoint
```suggestion
    @GetMapping("/{id}")
    public ResponseEntity<ExpenseDTO> findById(@PathVariable("id") Long id){
```